components/infobox/wikis/stormgate/infobox_map_custom.lua

```diff
@@ -0,0 +1,223 @@
+---
+-- @Liquipedia
+-- wiki=starcraft2
+-- page=Module:Infobox/Map/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local Operator = require('Module:Operator')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local Map = Lua.import('Module:Infobox/Map', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+local Title = Widgets.Title
+
+local CustomMap = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local CAMPS = {
+	{key = 'resourcecamps', name = 'Resource Camp'},
+	{key = 'speedcamps', name = 'Speed Camp'},
+	{key = 'healthcamps', name = 'Health Camp'},
+	{key = 'siegecamps', name = 'Siege Camp'},
```

Comment:
Maybe 
```suggestion
	{key = 'resourcecamps', name = 'Resource Camp(s)'},
	{key = 'speedcamps', name = 'Speed Camp(s)'},
	{key = 'healthcamps', name = 'Health Camp(s)'},
	{key = 'siegecamps', name = 'Siege Camp(s)'},
```
Not sure whether i really prefer it tho. Could omit the parentheses as well.